docs/principles/perceivable_principle.md

```diff
@@ -0,0 +1,193 @@
+[⬅️ Principles](../accessibility_principles.md)
+
+# 1. Perceivable principle
+
+Information and user interface components must be presentable to users in ways they can perceive.
+
+## 1.1 Text alternatives
+
+Provide text alternatives for any non-text content to be changed into other forms people need, such as large print, braille, speech, symbols, or more straightforward language.
+
+### 1.1.1 Non-text content (Level A)
+
+When talking about the screen elements, it is important to make information about all elements accessible and available. Some elements may not have text included by design or as a default user interface component. This can be seen when buttons are used with icons only, images, or other decorative elements.
+
+Some users may have issues identifying an element's functionality or description in general, and because of that, it is important to define non-text elements by accessibility features.
+
+> This guideline with provided techniques covers the *1.1.1 Non-text Content - Level A of the WCAG standard.*
+
+**Techniques**\
+[Android](../platforms/android/techniques_percievable_android.md#111-non-text-content-level-a), [iOS](../platforms/ios/techniques_perceivable_ios.md#111-non-text-content-level-a), [Flutter](../platforms/flutter/techniques_perceivable_flutter.md#111-non-text-content-level-a)
```

Comment:
Currently, the guideline description is platform agnostic, but success/failure techniques are split per platform.
@nikolamajcen Could we make the success/failure techniques also be platform agnostic?
If you read them, they do pretty much all say the same thing.

There is an issue of what to do with the code examples. Can we just remove them?
For example: 
The iOS techniques give an example on how to use the `addButton.accessibilityLabel = "Add"`, but at this point, the reader already knows how to add accessibility labels. He did read about it in the "Accessibility features on iOS" chapter.

Reply:
I get your point and in some scenarios this is fully true. Still, there are a lot of cases where criterias are not the same, and you cannot even make the same thing on all platforms.

I would keep them separated because the implementation should be a part of the criterias, because this is something that is missing on the web overall. And as an additional thing, we want to provide information that can be used on the project while reading the criterias - in some cases that is possible via implementation, and sometimes is not.

If we go fully "generic" with this, guidelines won't provide that value that we wanted in the first place.